src/App.js

```diff
@@ -24,17 +27,37 @@ class App extends Component {
     });
   }
 
+  getForecast(city){
+    fetchForecast(city)
+    .then((response) => {
+      if (this.refs.myRef) {
+        this.setState({
+          forecast: response,
+        });
+      }
+      console.log(response);
+    });  
+  }
+
   handleChange(city){
-    this.getWeather(city);    
+    this.getWeather(city);
+    this.getForecast(city);    
```

Comment:
Think this should go in a `Promise.all` way since I believe both are needed to update and render only once instead of twice